src/Controller/Component/ModulesComponent.php

```diff
@@ -670,6 +670,9 @@ protected function folderChildrenRelated(string $id, array $relatedIds)
             }
             // invert relation call => use 'parent' relation on children folder
             $data = compact('id') + ['type' => 'folders'];
+            if (Hash::check($item, 'meta')) {
```

Comment:
```suggestion
            if (Hash::check((array)$item, 'meta')) {
```
to make scrutinizer happy